### ISSUE TITLE
project sponsorship options not available

### ISSUE THREAD

**Issue by em-winterschon** [DtxdF/LittleJet#1]
I'd like to sponsor this project via either Patreon or Github.

**Comment by DtxdF** [comment]
Hi @em-winterschon 

Thank you very much for that. Unfortunately, my country, Venezuela, is currently not eligible for Github sponsors:

* https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#supported-regions-for-github-sponsors

However, there are some options available:

* [LiberaPay](https://liberapay.com/AppJail)
* [PayPal](https://paypal.me/DtxdF)
* [Patreon](https://patreon.com/AppJail)

But a few notes:

Since I can't use Stripe (again, my country), the only option I have to receive funding is PayPal, so LiberaPay is linked to it. I recently created Patreon and have some plans with it, but at the moment I'm just focusing on all the development of the projects. I recommend LiberaPay, it works very well.

Of course, I have shared the links for funding AppJail and this issue is in the LittleJet repository, but in case of funding it is the same as the big project is AppJail and all the meta projects (Director, Reproduce, Makejails, LittleJet , etc.) will be a benefit to them.
